disk.go

```diff
@@ -175,7 +184,8 @@ func (l *LatencyParabola) Latency(params DiskLatencyParams) time.Duration {
 
 func (d *Disk) scheduleWrite(sizebytes int) time.Duration {
 	at := sizeToDuration(sizebytes, "B", int64(d.MBps), "MB")
```

Comment:
Probably makes sense to subclass Disk type for the mb and future models. Older models can then keep using the basic Disk with its basic lastIOdone member variable. This would be safer. The changes in the scheduleWrite() and the need to maintain array of latencies would warrant this change..